rllib/algorithms/dqn/dqn_rainbow_learner.py

```diff
@@ -44,10 +41,8 @@ class DQNRainbowLearner(Learner):
     def build(self) -> None:
         super().build()
 
-        # Initially sync target networks.
-        self.module.foreach_module(
-            lambda mid, module: module.sync_target_networks(tau=1.0)
-        )
+        # Initially sync target networks (w/ tau=1.0 -> full overwrite).
+        self.module.sync_target_networks(tau=1.0)
```

Comment:
We sync twice at the beginning now - the `TorchDQNRainbowRLModule` does sync in its `setup()`.

Reply:
Good catch! I was debating this with myself: Should the RLModule perform the initial sync or the Learner?

Since the Learner also controls the regular syncs during training, I felt like we should do it in the Learner, then it's all in one place. The RLModule itself (at least in its inference_only mode) doesn't really care about the target nets anyways.

Reply:
fixed